pkg/operator/config.go

```diff
@@ -122,6 +124,9 @@ type Config struct {
 	// WebhookCertDir is the directory containing TLS assets for the admission
 	// webhook server.
 	WebhooksCertDir string
+
+	// metricsPort is the port the metrics are exposed.
+	metricsPort int
```

Comment:
Why would the metrics port be unexported unlike all the other fields?

Reply:
any reason for them to be exported? should we instead unexport in a different PR?

Reply:
They just seem like public fields to me. This is not something that's going to cross system boundaries. We don't have to be super protective of it, but if I want to pass it to another package within the project at some point, I should be able to read the fields.

Reply:
ok sg